data/hai/hai reveal 4 middle.txt

```diff
@@ -959,17 +959,17 @@ mission "Hai Reveal [A08] Symbol Identification"
 	to offer
 		has "Hai Reveal [A07] Deliver the Relays: done"
 	on visit
-		dialog `Batu is on another ship not yet in the system, take off and wait for the ship to arrive in order to continue.`
+		dialog `Batu is on another ship not yet in the system. Take off and wait for the ship to arrive in order to continue.`
 	on offer
 		log "People" "Giti Gilani" `Giti is a research librarian on Earth that specializes in the history of space flight and related archaic topics.`
-		log "Factions" "Base Principles" `A clandestine organization that existed within the Syndicate that was founded upon anarcho-syndicalist ideals. Its ranks eventually became bloated by enthusiastic recruits from Freedom that were keen to get rich on top of their freedom, and more anarchist than anarcho-syndicalist. The resulting tensions undermined the group, and Syndicate Internal Affairs scoured them from the organization.`
+		log "Factions" "Base Principles" `A clandestine organization that existed within the Syndicate that was founded upon anarcho-syndicalist ideals. Its ranks eventually became bloated by enthusiastic recruits from Freedom that were keen to get rich on top of their freedom, and were more anarchist than anarcho-syndicalist. The resulting tensions undermined the group, and Syndicate Internal Affairs scoured them from the organization.`
```

Comment:
I'm not sure about this one because it is in the past. I would also remove "on top of their freedom" because it is a repetition to the planet Freedom.